openedx/core/djangoapps/notifications/docs/notification_tray.md

```diff
@@ -0,0 +1,7 @@
+# How to enable the notification tray
+
+On the front end, the notification tray needs to be enabled to simplify the user experience.
+Users can click the bell icon in the header to access the notification tray, which will display notifications from the apps listed above.
+For detailed steps please view the following document (internal) [https://2u-internal.atlassian.net/wiki/x/YgBdMg](https://2u-internal.atlassian.net/wiki/x/YgBdMg)
+
+And explicit implementation of the notification tray on the Learning Dashboard can be viewed in the following document (internal): [https://2u-internal.atlassian.net/wiki/x/DoAmM](https://2u-internal.atlassian.net/wiki/x/DoAmM)
```

Comment:
@ayub02 @AhtishamShahid  can these internal documents be moved to public? @samuelallan72  we should remove the links if they are not public

Reply:
I'll create a new document in openEdx space.

Reply:
Here are the openEdx confluence docs.
[How to Enable Notification Tray](https://openedx.atlassian.net/wiki/x/AQARPQE)
[Notification Tray Implementation in Learner Dashboard](https://openedx.atlassian.net/wiki/x/GwA2PQE)